entity-view/impl/src/main/java/com/blazebit/persistence/view/impl/metamodel/ManagedViewTypeImpl.java

```diff
@@ -231,6 +235,14 @@ public ManagedViewTypeImpl(ViewMapping viewMapping, ManagedType<?> managedType,
                     requiredUpdatableAttributes.add(entry.getKey());
                 }
             }
+
+            Class<?> elementJavaType = getJavaType();
+            Class<?> mappedEntityClass = elementJavaType.getAnnotation(EntityView.class).value();
+            if ((mappedEntityClass.getModifiers() & Modifier.ABSTRACT) != 0 && (AnnotationUtils.findAnnotation(mappedEntityClass, MappedSuperclass.class) != null || AnnotationUtils.findAnnotation(mappedEntityClass, Entity.class) != null)) {
+                context.addError("Creatable EntityViews may not map abstract Entities. Mapped Entity: '" + elementJavaType.getName() + "'");
```

Comment:
Please also include the entity view name that causes the error as given by `javaType`.